examples/admin_delete_acls/admin_delete_acls.go

```diff
@@ -0,0 +1,142 @@
+// Delete ACLs
+package main
+
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+)
+
+// Parses a list of 7n arguments to a slice of n AclBindingFilter
+func parseAclBindingFilters(args []string) (aclBindingFilters kafka.AclBindingFilters, err error) {
+	nAclBindingFilters := len(args) / 7
+	parsedAclBindingFilters := make(kafka.AclBindingFilters, nAclBindingFilters)
+
+	for i := 0; i < nAclBindingFilters; i += 1 {
+		start := i * 7
+		resourceTypeString := args[start]
+		name := args[start+1]
+		resourcePatternTypeString := args[start+2]
+		principal := args[start+3]
+		host := args[start+4]
+		operationString := args[start+5]
+		permissionTypeString := args[start+6]
+
+		var resourceType kafka.ResourceType
+		var resourcePatternType kafka.ResourcePatternType
+		var operation kafka.AclOperation
+		var permissionType kafka.AclPermissionType
+
+		resourceType, err = kafka.ResourceTypeFromString(resourceTypeString)
+		if err != nil {
+			fmt.Printf("Invalid resource type: %s: %v\n", resourceTypeString, err)
+			return
+		}
+		resourcePatternType, err = kafka.ResourcePatternTypeFromString(resourcePatternTypeString)
+		if err != nil {
+			fmt.Printf("Invalid resource pattern type: %s: %v\n", resourcePatternTypeString, err)
+			return
+		}
+
+		operation, err = kafka.AclOperationFromString(operationString)
+		if err != nil {
+			fmt.Printf("Invalid operation: %s: %v\n", operationString, err)
+			return
+		}
+
+		permissionType, err = kafka.AclPermissionTypeFromString(permissionTypeString)
+		if err != nil {
+			fmt.Printf("Invalid permission type: %s: %v\n", permissionTypeString, err)
+			return
+		}
+
+		parsedAclBindingFilters[i] = kafka.AclBindingFilter{
+			Type:                resourceType,
+			Name:                name,
+			ResourcePatternType: resourcePatternType,
+			Principal:           principal,
+			Host:                host,
+			Operation:           operation,
+			PermissionType:      permissionType,
+		}
+	}
+	aclBindingFilters = parsedAclBindingFilters
+	return
+}
+
+func main() {
+
+	// 2 + 7n arguments to create n ACL binding filters
+	nArgs := len(os.Args)
+	aclBindingFilterArgs := nArgs - 2
+	if aclBindingFilterArgs <= 0 || aclBindingFilterArgs%7 != 0 {
+		fmt.Fprintf(os.Stderr,
+			"Usage: %s <broker> <resource-type1> <resource-name1> <resource-pattern-type1> "+
```

Comment:
Call it bootstrap-servers instead to avoid any confusion whether a particular broker must be used for admin ops